riscv/insns/c_fsw.h

```diff
@@ -1,7 +1,13 @@
 if (xlen == 32) {
-  require_extension(EXT_ZCF);
-  require_fp;
-  MMU.store<uint32_t>(RVC_RS1S + insn.rvc_lw_imm(), RVC_FRS2S.v[0]);
+  require_either_extension(EXT_ZCF, EXT_ZCMLSD);
+  if (p->extension_enabled(EXT_ZCF)) {
+    require_fp;
+    MMU.store<uint32_t>(RVC_RS1S + insn.rvc_lw_imm(), RVC_FRS2S.v[0]);
+  } else {  // c.sd - Store pair RV32
+    require_extension(EXT_ZCMLSD);
+    require(insn.rvc_rs2() % 2 == 0);
```

Comment:
Redundant with the `require` in READ_REG_PAIR; delete.

Reply:
done in 02fbb0d